### ISSUE TITLE
Original repo history

### ISSUE THREAD

**Issue by rainbyte** [esoeylemez/netwire#1]
Hi, I'm glad to see that netwire will be in development again :+1: 

I wanted to ask if it would be possible to include [original repo](http://hub.darcs.net/ertes/netwire) log within this one.
Mainly for historical and reference purposes.

Thanks in advance!

**Comment by esoeylemez** [comment]
Hi @rainbyte and thanks!

Better call it in _maintenance_ though rather than in _development_, which means that I will review and apply contributions and make sure that it compiles. Currently my development efforts go into the new wires library and some others.

The original repo itself has an incomplete history (≥ v5), and it's really not much of a _history_ to begin with (as a Darcs user I rewrote history a lot). So for simplicity's sake I decided to just ignore it. The old repo will stay online of course for historical reference.

Let's just leave this issue open for a while to see how other people feel about this, and if popular demand should arise I can create a historical branch.
